sci-biology/spm/ChangeLog

```diff
@@ -0,0 +1,7 @@
+# ChangeLog for sci-biology/spm
+# Copyright 1999-2015 Gentoo Foundation; Distributed under the GPL v2
+# $Header: $
+
+  17 Feb 2015;  <chymera@gentoo.org> +metadata.xml, +spm-8.ebuild.orig,
```

Comment:
Ok, so if I do decide to use `export`... as a temporary f what do I have to do after I run that? just `repoman commit` again? Also, where does repoman get my user data from? I heard it should be either `~/.gitconfig` or `./.git/config`. I guess the latter should not be cluttered with my user info, but the former states everything correctly. So what can I do to permanently tell repoman who I am?

chymera@zenbookhost ~/src/neurogentoo $ cat .git/config 
[core]
	repositoryformatversion = 0
	filemode = true
	bare = false
	logallrefupdates = true
[remote "origin"]
	url = https://github.com/TheChymera/neurogentoo.git
	fetch = +refs/heads/*:refs/remotes/origin/*
[branch "master"]
	remote = origin
	merge = refs/heads/master
[branch "spm-octave"]
	remote = origin
	merge = refs/heads/spm-octave
[branch "patch-1"]
	remote = origin
	merge = refs/heads/patch-1
[branch "patch-2"]
	remote = origin
	merge = refs/heads/patch-2
[branch "fsl"]
	remote = origin
	merge = refs/heads/fsl
[branch "afni"]
	remote = origin
	merge = refs/heads/afni
[branch "pythontex-bump"]
	remote = origin
	merge = refs/heads/pythontex-bump
[remote "official"]
	url = https://github.com/gentoo-science/sci.git
	fetch = +refs/heads/*:refs/remotes/official/*
chymera@zenbookhost ~/src/neurogentoo $ cat ~/.gitconfig 
[user]
	email = h.chr@mail.ru
	name = Horea Christian
	signingkey = C7DE3E5F
[credential]
	helper = cache --timeout=99999
[push]
	default = simple
chymera@zenbookhost ~/src/neurogentoo $ cat sci-biology/spm/ChangeLog 
# ChangeLog for sci-biology/spm
# Copyright 1999-2015 Gentoo Foundation; Distributed under the GPL v2
# $Header: $

  17 Feb 2015;  <chymera@gentoo.org> -spm-8.ebuild.orig, spm-8.ebuild:
  added quotation marks, removed ori file

  17 Feb 2015;  <chymera@gentoo.org> +metadata.xml, +spm-8.ebuild.orig,
  spm-8.ebuild:
  added metadata
```